test/test_cpp_extensions_aot.py

```diff
@@ -0,0 +1,162 @@
+import os
+import shutil
+import sys
+import unittest
+import warnings
+
+import torch.testing._internal.common_utils as common
+from torch.testing._internal.common_utils import IS_WINDOWS
+from torch.testing._internal.common_cuda import TEST_CUDA
+import torch
+import torch.backends.cudnn
+import torch.utils.cpp_extension
+
+try:
+    import torch_test_cpp_extension.cpp as cpp_extension
+    import torch_test_cpp_extension.msnpu as msnpu_extension
+except ImportError:
+    warnings.warn(
+        "test_cpp_extensions_aot.py cannot be invoked directly. Run "
+        "`python run_test.py -i test_cpp_extensions_aot` instead."
+    )
+
+
+class TestCppExtensionAOT(common.TestCase):
```

Comment:
Any substantive changes to this class? It looks like this is from the preexisting tests

Reply:
No substantive changes to this class; should be a copy-paste of the preexisting tests